docs/source/using/managing-minishift.adoc

```diff
@@ -157,10 +156,165 @@ $ minishift config get memory
 To remove a persistent configuration option, you can use the
 link:../command-ref/minishift_config_unset{outfilesuffix}[`unset`] sub-command. For example:
 
+[source,sh]
 ----
 $ minishift config unset memory
 ----
 
+[[addons]]
+== Addons
+
+Minishift allows to configure the vanilla OpenShift setup as provided by _cluster up_ via a so called
+addon mechanism.
+
+Addons are directories, containing at least one text file with the _addon_ suffix. There can be other
```

Comment:
Instead of "suffix" , how about "extension" ? 
I think "extension" is more known than "suffix" .